src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
```

Comment:
It's a good practice to avoid creating a new instance of Random every time the method is called. Instead, you should create a single instance of Random as a class-level field and reuse it.